src/components/List.js

```diff
@@ -1,5 +1,27 @@
 import React from 'react';
+import { usePages } from '../hooks/usePages';
+import Page from './Page';
 
-const List = () => <h1>List</h1>;
+const List = () => {
+  const centerMap = {
+    // lat: 0,
+    // lng: 0,
+    lat: 48.8566,
+    lng: 2.3522,
+  };
+  const { pages, loading } = usePages(centerMap);
 
+  return (
+    <div>
+      <h1>List</h1>
+      <ul className="pages">
+        {loading && <p>Loading... </p>}
```

Comment:
also, do we need to handle the error case where the user doesn't have network access? maybe there's something from the map you can borrow to implement that?

Reply:
If we don't have results, we are going to show a message.

![image](https://user-images.githubusercontent.com/8689897/126830190-a98f872d-8c1c-4f06-9f50-83b1616258a1.png)